package.json

```diff
@@ -69,7 +69,7 @@
     "karma-mocha": "^2.0.0",
     "karma-remap-coverage": "^0.1.5",
     "karma-sourcemap-loader": "^0.4.0",
-    "karma-webpack": "^5.0.0",
+    "karma-webpack": "^4.0.0",
```

Comment:
This is unrelated, `webpack` and `karma-webpack` v5 require Node 18 and it breaks tests. These are dev dependencies, we don't really care.